### PR TITLE
Add `FULLTEXT` filtering to `SHOW INDEXES`

### PR DESCRIPTION
**Pull request by Hunterness**
~~Currently have broken tests due to relationship property indexes now being blocked (https://github.com/neo-technology/neo4j/pull/9567), fixed by https://github.com/neo4j/neo4j-documentation/pull/1069.~~

### CODE REVIEW

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala [hunk 1]
```diff
@@ -186,8 +186,11 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
       title = "Listing indexes with filtering",
       text =
         """
-          |One way of filtering the output from `SHOW INDEXES` is to use the `BTREE` keyword to only list `BTREE` indexes.
-          |Another is to use the `WHERE` clause, for example to only show indexes not belonging to constraints.""".stripMargin,
+          |One way of filtering the output from `SHOW INDEXES` by index type is the type keywords,
```
Comment (SaschaPeukert) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala]:
```suggestion
          |One way of filtering the output from `SHOW INDEXES` by index type is the use of type keywords,
```

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala [hunk 2]
```diff
@@ -186,8 +186,11 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
...
+          |listed in the <<administration-indexes-syntax, syntax table>>.
+          |As an example, to show only B-tree indexes, use `SHOW BTREE INDEXES`.
+          |Another, more flexible way of filtering the output is to use the `WHERE` clause.
+          |For example to only show indexes not belonging to constraints.""".stripMargin,
```
Comment (SaschaPeukert) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala]: This sentence sounds like its not finished. Like its leading to an example that does not follow

**Review by SaschaPeukert — CHANGES_REQUESTED** [review]

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala (hunk 1 above)
Comment (Hunterness) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala]: should update that for constraints as well then, think I just took the sentence from there

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala (hunk 2 above)
Comment (Hunterness) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala]: There is supposed to be an example after? (the query below?)

Comment (SaschaPeukert) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala]: Thats my point. The query below does not fit the text "only show indexes not belonging to constraints", right?

Comment (Hunterness) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala]: - ```As an example, to show only B-tree indexes, use `SHOW BTREE INDEXES`.```
- `For example to only show indexes not belonging to constraints.`

`SHOW BTREE INDEXES WHERE uniqueness = 'NONUNIQUE'` shows all BTREE indexes not belonging to any constraints

Comment (Hunterness) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala]: the example combines both filtering versions

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala (hunk 1 above)
Comment (Hunterness) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala]: added to the constraints PR

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala (hunk 2 above)
Comment (SaschaPeukert) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala]: I am legit okay with any of the two as they both don't seem to reference the next query :-)

Comment (Hunterness) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala]: not sure what you mean?

the query in the example is `SHOW BTREE INDEXES WHERE uniqueness = 'NONUNIQUE'` with
the WHERE clause is filtering on indexes without constraints and the BTREE keyword filters on type which is the 2 things the text describes?

Comment (SaschaPeukert) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala]: Ah okay, slight miscommunication, now I understand :-D So my problem is that the last sentence does not sound like its belonging to the rest of the paragraph but instead sounds like it's something new. And if you read it like that, then the query underneath does not fit.

**Review by SaschaPeukert — APPROVED** [review]

**Comment by Hunterness** [comment]
Will be included in https://github.com/neo4j/neo4j-documentation/pull/1075

**Pull request closed without merging**